package.json

```diff
@@ -131,7 +131,7 @@
     "our:verify:prettier": "prettier --write ./**",
     "our:verify:stylelint": "echo 'disabled due to https://spectrum.chat/zeit/general/resolved-deployments-fail-with-enospc-no-space-left-on-device-write~d1b9f61a-65e8-42a3-9042-f9c6a6fae6fd'",
     "our:verify:typescript": "tsc --noEmit --project tsconfig.json",
-    "env": "env NODE_ENV=development GOOGLE_OAUTH=\"$(cat ./.env/google-oauth.json | jq -c .)\" GOOGLE_SERVICE=\"$(cat ./.env/google-service.json | jq -c .)\"",
+    "env": "env NODE_ENV=development FAUNADB=\"$(cat ./.env/faunadb.json | jq -c .)\" GOOGLE_OAUTH=\"$(cat ./.env/google-oauth.json | jq -c .)\" GOOGLE_SERVICE=\"$(cat ./.env/google-service.json | jq -c .)\"",
```

Comment:
rename `FAUNADB` here to `FAUNA_SERVER_KEY`

Reply:
faunadb.json contains FAUNA_SERVER_KEY

{
    "FAUNADB_SECRET_KEY": "fnADcwOLnuACAsblpy_dr7hASiEsFxBU_bk18CrG"
}

Would FAUNADB_SERVICE, FAUNA_SERVICE work better here? I was just taking clue from GOOGLE_OAUTH and GOOGLE_SERVICE so I put all the fauna env settings in faunadb.json so it will contain fauna server key but any other fauna related env settting as well so not a good idea to name it as FAUNA_SERVER_KEY.